pkg/sql/importer/import_job.go

```diff
@@ -1502,22 +1503,42 @@ func (r *importResumer) dropTables(
 		// writes, so even if GC has run it would not have GC'ed any keys to which
 		// we need to revert, so we can safely ignore the target-time GC check.
 		const ignoreGC = true
-		if err := sql.RevertTables(ctx, txn.DB(), execCfg, revert, ts, ignoreGC, sql.RevertTableDefaultBatchSize); err != nil {
+		if err := sql.RevertTables(ctx, txn.DB(), execCfg, []catalog.TableDescriptor{intoTable}, ts, ignoreGC,
+			sql.RevertTableDefaultBatchSize); err != nil {
 			return errors.Wrap(err, "rolling back partially completed IMPORT")
 		}
 	}
 
-	for i := range empty {
-		// Set a DropTime on the table descriptor to differentiate it from an
-		// older-format (v1.1) descriptor. This enables ClearTableData to use a
-		// RangeClear for faster data removal, rather than removing by chunks.
-		empty[i].TableDesc().DropTime = dropTime
-		if err := gcjob.ClearTableData(
-			ctx, execCfg.DB, execCfg.DistSender, execCfg.Codec, &execCfg.Settings.SV, empty[i],
-		); err != nil {
-			return errors.Wrapf(err, "clearing data for table %d", empty[i].GetID())
-		}
+	// Set a DropTime on the table descriptor to differentiate it from an
+	// older-format (v1.1) descriptor. This enables ClearTableData to use a
+	// RangeClear for faster data removal, rather than removing by chunks.
+	intoTable.TableDesc().DropTime = int64(1)
+	if err := gcjob.ClearTableData(
+		ctx, execCfg.DB, execCfg.DistSender, execCfg.Codec, &execCfg.Settings.SV, intoTable,
+	); err != nil {
+		return errors.Wrapf(err, "clearing data for table %d", intoTable.GetID())
+	}
+
+	// Bring the IMPORT INTO table back online
+	b := txn.NewBatch()
+	intoDesc, err := descsCol.GetMutableTableVersionByID(ctx, intoTable.GetID(), txn)
+	if err != nil {
+		return err
 	}
+	intoDesc.SetPublic()
+	const kvTrace = false
+	if err := descsCol.WriteDescToBatch(ctx, kvTrace, intoDesc, b); err != nil {
+		return err
+	}
+	return errors.Wrap(txn.Run(ctx, b), "putting IMPORT INTO table back online")
+}
+
+//dropNewTables rolls back an IMPORT cmd
```

Comment:
nit: flesh out comment to something like:

dropNewTables drops the tables that were created as part of an `IMPORT` and queues a GC job to cleanup the dropped descriptors.

Reply:
Done.